src/meta/src/hummock/manager/time_travel.rs

```diff
@@ -52,6 +53,11 @@ impl HummockManager {
     }
 
     pub(crate) async fn init_time_travel_state(&self) -> Result<()> {
+        if self.env.opts.enable_hummock_time_travel {
```

Comment:
I think gating in the frontend might be better? Since it can be more responsive in case users update their license keys online.